scripts/download-vscode.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+const ncp = require('ncp');
+const path = require('path');
+const fs = require('fs');
+const downloadAndUnzipVSCode = require('vscode-test').downloadAndUnzipVSCode;
+
+// Copy vscode to these directories
+const extensionDirectories = [
+  'salesforcedx-vscode-apex',
+  'salesforcedx-vscode-apex-debugger',
+  'salesforcedx-vscode-apex-replay-debugger',
+  'salesforcedx-vscode-core',
+  'salesforcedx-vscode-lightning',
+  'salesforcedx-vscode-lwc',
+  'salesforcedx-vscode-visualforce'
+];
+
+// Executable path looks something like:
+// ~/salesforcedx-vscode/packages/salesforcedx-vscode-lwc/.vscode-test/vscode-1.41.1/Visual Studio Code.app/Contents/MacOS/Electron
+downloadAndUnzipVSCode()
+  .then(executablePath => {
+    console.log('Executable Path: ' + executablePath);
+
+    let vscodeIndex = executablePath.indexOf('.vscode-test') + 13;
+
+    // 'vscode-1.41.1'
+    let vscodeDirname = executablePath.substring(
+      vscodeIndex,
+      executablePath.indexOf(path.sep, vscodeIndex)
+    );
+    process.env.CODE_VERSION = vscodeDirname;
```

Comment:
This doesn't work, we'll have to check the copied directory name to get the version